html/index.html

```diff
@@ -0,0 +1,261 @@
+		
+                <b> 
+            <a id="a1" href="index.htm"> HOME </a>  *
+
+            <a id="a2" href="#"> ABOUT US </a> *
+
+            <a id="a3" href="http://www.subaru.ua/dealers"> CONTACT US </a> * 
+
+            <a id="a4" href="#"> SERVICE </a> *
+                </b> </td> 
+	</tr>
+
+	<tr>
+		<tr colspan=8>
+			<img id="l6" src="images/layout_06.gif"></td>
+	</tr>
+	<tr>
+		<td colspan=6>
+			<img id="l7" src="images/layout_07.gif"></td>
+		<td colspan=8>
+			<img id="l8" src="images/layout_08.gif"></td>
+	</tr>
+	<tr>
+		<td colspan=6>
+			<img id="l9" src="images/layout_09.gif"></td>
+		<td>
+			<img id="l10" src="images/layout_10.gif"></td>
+		<td>
+			<img id="l11" src="images/layout_11.gif"></td>
+		<td>
+			<img id="l12" src="images/layout_12.gif"></td>
+		<td>
+			<img id="l13" src="images/layout_13.gif"></td>
+		<td>
+			<img id="l14" src="images/layout_14.gif"></td>
+		<td>
+			<img id="l15" src="images/layout_15.gif"></td>
+		<td>
+			<img id="l16" src="images/layout_16.gif"></td>
+		<td rowspan=2>
+			<img id="l17" src="images/layout_17.gif"></td>
+	</tr>
+	<tr>
+		<td colspan=6>
+			<img id="l18" src="images/layout_18.gif"></td>
+		<td colspan=7>
+			<img id="l19" src="images/layout_19.gif"></td>
+	</tr>
+	<tr>
+		<td colspan=3 rowspan="2" valign="top" background="images/layout_32.gif">
+
+	
+<table id="t2">
+              <tr>
+                <td colspan="3">
+                <img id="l20" src="images/layout_20.gif"></td>
+              </tr>
+
+              <tr>
+                <td background="images/layout_23.gif">
+                <img id="l23" src="images/layout_23.gif"></td>
+                <td background="images/layout_24.gif" valign="top">
+                
+                <font color="#000000"><b margin-top="0">31/07/2015 <br>
+                </b>We offer great 20% discount on Subaru Forester till 15/08/2015 !</font> <br> </td>
+                <td background="images/layout_25.gif">
+                <img id="l25" src="images/layout_25.gif"></td>
+              </tr>
+              <tr>
+                <td colspan="3">
+                <img id="l27" src="images/layout_27.gif"></td>
+              </tr>
+              <tr>
+                <td background="images/layout_28.gif">
+                <img id="l28" src="images/layout_28.gif"></td>
+                <td background="images/layout_24.gif" valign="top">
+                <img id="l29" src="images/layout_29.gif"><br>
+                <font color="#000000"><b>30/07/2015<br>
+                </b>Today we celebrate our 15th anniversary! Happy BirthDay, Car Company! </font><br> </td>
+                <td background="images/layout_30.gif">
+                <img id="l30" src="images/layout_30.gif" ></td>
+              </tr>
+              <tr>
+                <td colspan="3">
+                <img id="l31" src="images/layout_31.gif"></td>
+              </tr>
+            </table>
+			<img id="l32" src="images/layout_32.gif" ></td>
```

Comment:
http://joxi.ru/Q2K0RDnse8B6Aj

Have you even tested you html? 
your images are stored in "vendor" folder. But your src is pointing to "images" folder.